app/views/hq/orphans/show.html.haml

```diff
@@ -0,0 +1,121 @@
+.panel.panel-default
+  .panel-heading 
+    %h3.panel-title Orphan Details
+  .panel-body
+    %dl.dl-horizontal
+      %dt OSRA Num
+      %dd
+        = @orphan.osra_num
+    %dl.dl-horizontal
+      %dt Orphan Name
+      %dd
+        = @orphan.name
+    %dl.dl-horizontal
+      %dt Date of Birth
+      %dd
+        = format_full_date @orphan.date_of_birth
+    %dl.dl-horizontal
+      %dt Gender
+      %dd
+        = @orphan.gender
+    %dl.dl-horizontal
+      %dt Contact Number
+      %dd
+        = @orphan.contact_number
+    %dl.dl-horizontal
+      %dt Orphan Status
+      %dd
+        = @orphan.orphan_status.name
+    %dl.dl-horizontal
+      %dt Sponsorship Status
+      %dd
+        = @orphan.orphan_sponsorship_status.name
+    %dl.dl-horizontal
+      %dt Priority
+      %dd
+        = @orphan.priority
+.panel.panel-default
+  .panel-heading 
+    %h3.panel-title Parent's Details
+  .panel-body
+    %dl.dl-horizontal
+      %dt Father Given Name
```

Comment:
Let's take this opportunity to correct the app's grammar: "Father's Given Name" & "Mother's Given Name". This has bugged me for a long time :)